packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -56,6 +56,11 @@ export interface ISemanticColors {
    */
   bodyBackground: string;
 
+  /**
+   * The standout color for backgrounds.
```

Comment:
Describe briefly that standout is highlighted content, and that it should be using the neutral variant when on no emphasis, and that it's a shade darker the other emphases.